scripts/common.lib.ps1

```diff
@@ -102,46 +102,34 @@ function Locate-PackagesPath {
   return Resolve-Path -path $packagesPath
 }
 
-function Locate-VsInstallPath($hasVsixExtension = "false") {
-   $locateVsApi = Locate-LocateVsApi
-   $requiredPackageIds = @()
-
-   $requiredPackageIds += "Microsoft.Component.MSBuild"
-   $requiredPackageIds += "Microsoft.Net.Component.4.6.TargetingPack"
-   $requiredPackageIds += "Microsoft.VisualStudio.Component.Roslyn.Compiler"
-   
-   if($hasVsixExtension -eq 'true')
-   {
-     $requiredPackageIds += "Microsoft.VisualStudio.Component.VSSDK"
-   }
-   
-   Write-Verbose "VSInstallation requirements : $requiredPackageIds"
-
-   Add-Type -path $locateVsApi
-
-   Try
-   {
-     $vsInstallPath = [LocateVS.Instance]::GetInstallPath($msbuildVersion, $requiredPackageIds)
-   }
-   Catch [System.Management.Automation.MethodInvocationException]
-   {
-      Write-Error "Failed to find VS installation with requirements : $requiredPackageIds"
-   }
+function Locate-VsWhere {
+	$nuget = Locate-NuGet
+	
+	Write-Verbose "$nuget install vswhere -OutputDirectory $env:TF_ROOT_DIR -ExcludeVersion | Out-Null"
+	& $nuget install vswhere -OutputDirectory $env:TF_ROOT_DIR -ExcludeVersion | Out-Null
+	
+	$vswhere = Join-Path -path $env:TF_ROOT_DIR -childPath "vswhere\tools\vswhere.exe"
```

Comment:
We should remove `RoslynTools.Microsoft.LocateVS` from packages.config.
Should we use packages.config to get vswhere? I think it will be consistent with rest of dependencies.